### PR TITLE
[CI:DOCS] man pages: replace -c with --cpu-shares

### PR DESCRIPTION
**Pull request by eriksjolund**
These three man pages have some similar text.

https://github.com/containers/podman/blob/edbfbfcda18f57c89a9657ba453c51977a7796a9/docs/source/markdown/podman-run.1.md?plain=1#L247-L256

https://github.com/containers/podman/blob/edbfbfcda18f57c89a9657ba453c51977a7796a9/docs/source/markdown/podman-create.1.md?plain=1#L231-L239

https://github.com/containers/podman/blob/edbfbfcda18f57c89a9657ba453c51977a7796a9/docs/source/markdown/podman-container-clone.1.md?plain=1#L91-L99

 I assume podman-run.1.md is the correct one (i.e. using __--cpu-shares=__ instead of __-c=__)

Signed-off-by: Erik Sjölund <erik.sjolund@gmail.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by rhatdan** [comment]
-c and --cpu-shares are the same, although having these use --cpu-shares is easier for humans to understand.
LGTM

**Comment by eriksjolund** [comment]
It seems the man pages don't mention __-c__

https://github.com/containers/podman/blob/edbfbfcda18f57c89a9657ba453c51977a7796a9/docs/source/markdown/podman-run.1.md?plain=1#L223

https://github.com/containers/podman/blob/edbfbfcda18f57c89a9657ba453c51977a7796a9/docs/source/markdown/podman-container-clone.1.md?plain=1#L64

https://github.com/containers/podman/blob/edbfbfcda18f57c89a9657ba453c51977a7796a9/docs/source/markdown/podman-create.1.md?plain=1#L204

**Comment by eriksjolund** [comment]
__-c__ is not mentioned here either
```
$ podman create --help| grep cpu-sh
      --cpu-shares uint                          CPU shares (relative weight)
$ podman run --help| grep cpu-sh
      --cpu-shares uint                          CPU shares (relative weight)
$ podman container clone --help| grep cpu-sh
      --cpu-shares uint                       CPU shares (relative weight)
$ 
```

**Comment by rhatdan** [comment]
Indeed you are right. Looks like you have found a compatibility issue.

```
$ podman create --help | grep -- --cpu-shares
      --cpu-shares uint                          CPU shares (relative weight)
$ docker create --help | grep -- --cpu-shares
  -c, --cpu-shares int                 CPU shares (relative weight)
```

File: docs/source/markdown/podman-container-clone.1.md [hunk 1]
```diff
@@ -88,15 +88,16 @@ On a multi-core system, the shares of CPU time are distributed over all CPU
 cores. Even if a container is limited to less than 100% of CPU time, it can
 use 100% of each individual CPU core.
 
-For example, consider a system with more than three cores.
-container **{C0}** is started with **-c=512** running one process, and another container
-**{C1}** with **-c=1024** running two processes, this can result in the following
+For example, consider a system with more than three cores. If you start one
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-container-clone.1.md:91]: no pronouns in man pages please

Instead "If the container **(Co}** is started with..."

Comment (TomSweeneyRedHat) [reply docs/source/markdown/podman-container-clone.1.md:91]: And I see the "you" bit is older in other man pages,  Please rework those too and apologies for the added headache.

@edsantiago in your marvelous man page parser, could we add a check for the word "you" and flag that?

**Comment by TomSweeneyRedHat** [comment]
@edsantiago sorry, I wasn't clear.  I wasn't looking to get them all removed at this point, a fun weekend project for me.  What I was wondering if there's a new " you " added in PR for the man pages, just the ones that live in https://github.com/containers/podman/tree/main/docs/source/markdown, then the CI would throw a warning or perhaps fail.

Pronouns in readme's, tutorials, and other docs are fine.  We just want to keep the man pages crisp and sans pronouns, or at least " you ".

**Comment by edsantiago** [comment]
That would require running `git diff`, which is a big can of worms. I'm sorry.

**Comment by TomSweeneyRedHat** [comment]
@edsantiago no worries at all, I was hoping it might be quick and easy.  Thanks!

File: docs/source/markdown/podman-container-clone.1.md (hunk 1 above)
Comment (eriksjolund) [reply docs/source/markdown/podman-container-clone.1.md]: I removed the pronoun. I also removed the strange curly brace around the container name.

**Comment by rhatdan** [comment]
@eriksjolund You have whitespace issues, and then we can merge.

**Comment by eriksjolund** [comment]
whitespace now removed

**Pull request merged**